src/components/aws/accounts/teamSharing/FormComponent.js

```diff
@@ -0,0 +1,177 @@
+    }
+  }
+
+  openDialog = (e) => {
+    e.preventDefault();
+    this.setState({
+      open: true
+    });
+    this.props.clear();
+  };
+
+  closeDialog = (e) => {
+    e.preventDefault();
+    this.setState({open: false});
+    this.props.clear();
+  };
+
+  setPermission(permission) {
+    this.setState({permission: parseInt(permission, 10)});
+  }
+
+  submit = (e) => {
+    e.preventDefault();
+    const formValues = this.form.getValues();
+    this.props.submit(formValues.email, this.state.permission);
+  };
+
+  render() {
+    const loading = (this.props.status && !this.props.status.status ? (<Spinner className="spinner clearfix" name='circle'/>) : null);
+
+    const error = (this.props.status && this.props.status.status && this.props.status.hasOwnProperty("error") ? (
+      <div className="alert alert-warning" role="alert">{this.props.status.error.message}</div>
+    ) : null);
+
+    return (
+      <div>
+
+        <button className="btn btn-default" onClick={this.openDialog}>
+          {this.props.AccountViewer !== undefined ? <i className="fa fa-edit"/> : <i className="fa fa-plus"/>}
+          &nbsp;
+          {this.props.AccountViewer !== undefined ? "Edit" : "Invite a user"}
+        </button>
+
+        <Dialog open={this.state.open} fullWidth>
+
+          <DialogTitle disableTypography><h1>
+            <i className="fa fa-users red-color"/>
+            &nbsp;
+            {this.props.AccountViewer !== undefined ? "Edit this shared user" : "Invite a user"}
+          </h1></DialogTitle>
+
+          <DialogContent>
+
+            {loading || error}
+
+            <Form ref={
+                /* istanbul ignore next */
+                form => { this.form = form; }
+              }
+              className="team-sharing-form"
+              onSubmit={this.submit}>
+
+
+              <div className="form-group">
+                <div className="input-title">
+                  <label htmlFor="guestmail">Mail address</label>
+                </div>
+                <Input
+                  name="email"
+                  type="email"
+                  className="form-control"
+                  placeholder="john.doe@domain.com"
+                  value={this.props.AccountViewer ? this.props.AccountViewer.email : undefined}
+                  disabled={this.props.AccountViewer !== undefined}
+                  onChange={this.handleInputChange}
+                  validations={[Validation.required, Validation.guestMailFormat]}
+                />
+              </div>
+
+              <div className="form-group">
+                <div className="input-title">
+                  <label htmlFor="guestpermission">Permission Level</label>
+                  &nbsp;
+                  <Popover info tooltip="Choose carefully : Administrator can edit everything"/>
+                </div>
+                <Selector
+                  values={permissions}
+                  selected={this.state.permission}
```

Comment:
This raises a prop types error : 
![screenshot 2018-09-27 at 15 33 28](https://user-images.githubusercontent.com/3480526/46151208-92ce8800-c26e-11e8-9380-30b36ce3ae2c.png)